app/scripts/modules/core/src/managed/config/ManagementWarning.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { showModal } from 'core/index';
+
+import { ResumeManagementModal } from './Configuration';
+import { useFetchApplicationManagementStatusQuery, useToggleManagementMutation } from '../graphql/graphql-sdk';
+import { MODAL_MAX_WIDTH } from '../utils/defaults';
+import './ManagementWarning.less';
+
+export const ManagementWarning = ({ appName }: { appName: string }) => {
+  const { data, refetch } = useFetchApplicationManagementStatusQuery({ variables: { appName } });
+  const [toggleManagement] = useToggleManagementMutation();
+
+  if (data?.application?.isPaused) {
+    return (
+      <div className="ManagementWarning">
+        <div className="content sp-padding-m">
+          Application management is disabled.{' '}
+          <a
+            href="#"
+            onClick={(e) => {
+              e.preventDefault();
+              showModal(
+                ResumeManagementModal,
+                {
+                  application: appName,
+                  onAction: async () => {
+                    await toggleManagement({ variables: { application: appName, isPaused: false } });
+                    refetch();
+                  },
+                  logCategory: 'App::Management',
+                  onSuccess: refetch,
+                  withComment: false,
+                },
+                { maxWidth: MODAL_MAX_WIDTH },
+              );
+            }}
```

Comment:
For readability, can you pull this out into a named function? `onClick={showManagementModal}`